app/domain/operations/eligible/migration_csv_export.rb

```diff
@@ -88,7 +88,7 @@ def process_entry(csv, entry)
             record["evidences"].each do |evidence|
               csv << (
                 [
-                  subject.hbx_id,
+                  (subject.try(:hbx_id) || subject.id),
```

Comment:
@nks2109  - expectation is to always have subject here; i've used `try` here b'coz we're using this migration for different subjects and we don't have hbx_id in case of bqt and it's erroring out.